stout_a07b.js

```diff
@@ -0,0 +1,101 @@
+/*Kyle Stout
+  stout_a02b
+  INFO 2124WW
+  Thoendel
+  02-02-2020
+*/
+
+/* DO NOT MODIFY CODE BETWEEN THESE LINES */
+const standardInput = process.stdin;            
+standardInput.resume();
+standardInput.setEncoding('utf8');
+const badFoods = [];
+const groceryItems = [];
+clearScreen();
+console.log(getPrompt(groceryItems));

```

Comment:
notice that we are logging whatever is returned here - so getPrompt should return a string....not log directly.